assets/src/stories-editor/components/context-menu/index.js

```diff
@@ -155,7 +155,9 @@ const ContextMenu = ( props ) => {
 
 		const rootClientId = getBlockRootClientId( clientId );
 		const clonedBlock = cloneBlock( block );
-		insertBlock( clonedBlock, null, rootClientId );
+		const parentBlock = ( clonedBlock.name === 'amp/amp-story-page' ) ? '' : rootClientId;
+
+		insertBlock( clonedBlock, null, parentBlock );
```

Comment:
If we duplicate a Page, should we perhaps also position it next to the origin Page instead?

Reply:
Is this possible?

Reply:
Yep, instead of `null` we could use `currentPageIndex + 1`, e.g.
`insertBlock( clonedBlock, index, '' );`

Reply:
(By current Page index I mean the origin Page index that's being duplicated.)

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3537/commits/94e27aa541868fe57ee96ae8067f5619c23872f1